src/components/behandlerdialog/meldinger/MeldingerISamtale.tsx

```diff
@@ -49,13 +53,23 @@ interface MeldingInnholdProps {
 }
 
 const MeldingFraBehandler = ({ melding }: MeldingInnholdProps) => {
+  const { isFeatureEnabled } = useFeatureToggles();
+  const isReturLegeerklaringEnabled = isFeatureEnabled(
+    ToggleNames.behandlerdialogReturLegeeklaring
+  );
+  const isLegeerklaring =
+    melding.type === MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING;
+
   return (
     <StyledMelding innkommende>
       <StyledImageWrapper innkommende>
         <img src={StetoskopIkonBakgrunn} alt="Stetoskopikon for behandler" />
       </StyledImageWrapper>
       <StyledInnhold>
         <MeldingInnholdPanel melding={melding} />
+        {isReturLegeerklaringEnabled && isLegeerklaring && (
```

Comment:
Mulig det må sjekkes mer her etter hvert... Skal det f.eks. være mulig å sende retur flere ganger på samme legeerklæring?

Reply:
Ja, du tenker altså hvis den fortsatt ikke er god nok etter første retur, og man ønsker enda en forbedret legeerklæring? Mulig man vil det, må kanskje spørre Stine/TH!

Reply:
Ja, det er jo en ting også. 
Men sånn logikken er nå så vil jo knappen alltid ligge der for en mottatt legeerklaring så man kan i teorien sende retur-melding mange ganger på den samme.